services/web/server/src/simcore_service_webserver/projects/db.py

```diff
@@ -444,6 +444,68 @@ async def get_project(
                 project_type,
             )
 
+    # NOTE: MD: I intentionally didn't include the workbench. There is a special interface
+    # for the workbench, and at some point, this column should be removed from the table.
+    # The same holds true for access_rights/ui/classifiers/quality, but we have decided to proceed step by step.
+    _SELECTION_PROJECT_DB_ARGS = [
+        projects.c.id,
+        projects.c.type,
+        projects.c.uuid,
+        projects.c.name,
+        projects.c.description,
+        projects.c.thumbnail,
+        projects.c.prj_owner,
+        projects.c.creation_date,
+        projects.c.last_change_date,
+        projects.c.access_rights,
+        projects.c.ui,
+        projects.c.classifiers,
+        projects.c.dev,
+        projects.c.quality,
+        projects.c.published,
+        projects.c.hidden,
+    ]
+
+    async def get_project_db(self, project_uuid: ProjectID) -> ProjectDB:
+        async with self.engine.acquire() as conn:
+            result = await conn.execute(
+                sa.select(*self._SELECTION_PROJECT_DB_ARGS).where(
+                    projects.c.uuid == f"{project_uuid}"
```

Comment:
Q: in your opinion, what is the role of the project.id (i.e. int) as primary key since we have project.uuid . Should we drop the former and set the latter as primary key?

Reply:
Hmm, valid question. If we were to use the project ID (the primary key) in all the database repositories and only use the UUID for the frontend or where it's needed, it would be okay. However, since we use the UUID everywhere as if it were the primary key, we can remove the project ID. This might also slightly improve query performance.